README.md

```diff
@@ -8,14 +8,15 @@ An example workflow that uses [GitHub Actions](https://help.github.com/en/catego
 1. Set up Flux on the cluster, e.g. using [this guide](https://docs.fluxcd.io/en/latest/tutorials/get-started.html). Note that you must set `--git-path` to point to where your manifests are. For example:
 ```bash
 export GHUSER=<github user account where your fork lives>
```

Comment:
Good thought, but could we instead rename `GHUSER` to `GHOWNER` to also include organizations?
```suggestion
export GHOWNER=<github user or organization account where your fork lives>
```